Loop/View Controllers/GlucoseThresholdTableViewController.swift

```diff
@@ -21,9 +21,9 @@ final class GlucoseThresholdTableViewController: TextFieldTableViewController {
         
         super.init(style: .grouped)
         
-        placeholder = NSLocalizedString("Enter suspend threshold", comment: "The placeholder text instructing users to enter a suspend treshold")
+        placeholder = NSLocalizedString("Enter glucose safety limit", comment: "The placeholder text instructing users to enter a suspend treshold")
```

Comment:
[nit] you updated the comment in `BolusRecommendation.swift` but not here.  Consider being consistent and update the comments everywhere.